### PR TITLE
WIP: Add tile pointing to build repo

### PR DESCRIPTION
**Pull request by barthalion**

### CODE REVIEW

**Comment by razzeee** [comment]
I've updated this, it should work now.

I'm not sure how happy I'm with `Manifest` for this

File: src/types/Appstream.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ export interface Appstream {
   provides: string[]
   launchable: Launchable
   bundle: Bundle
+  metadata: Metadata
```
Comment (bilelmoussaoui) [inline src/types/Appstream.ts:19]: Shouldn't this be metadata?: Metadata instead

Comment (razzeee) [reply src/types/Appstream.ts]: Yeah, seems better.

File: src/types/Appstream.ts [hunk 2]
```diff
@@ -105,3 +106,7 @@ export interface Bundle {
   runtime: string
   sdk: string
 }
+
+export interface Metadata {
+  'Flathub::manifest': string
```
Comment (bilelmoussaoui) [inline src/types/Appstream.ts:111]: this should also be 
```suggestion
  'Flathub::manifest'?: string
```
because it is optional iirc

Comment (barthalion) [reply src/types/Appstream.ts:111]: Addressed in a separate commit just now.

**Pull request merged**